metabulo/models.py

```diff
@@ -29,8 +30,9 @@
 })
 db = SQLAlchemy(metadata=metadata)
 
-TABLE_COLUMN_TYPES = ['key', 'metadata', 'metabolite', 'masked']
-TABLE_ROW_TYPES = ['header', 'metadata', 'sample', 'masked']
+TableTypes = namedtuple('TableTypes', ['INDEX', 'METADATA', 'DATA', 'MASK'])
+TABLE_COLUMN_TYPES = TableTypes('key', 'metadata', 'metabolite', 'masked')
+TABLE_ROW_TYPES = TableTypes('header', 'metadata', 'sample', 'masked')
```

Comment:
if `len(column types) != len(row types)` this is easy to fix right?  We can just break the named tuples into row and col.

Reply:
Could we trade `metabolite` and `metabolites` as strings and variable names for `measurement` and `measurements`?

I can do this in a separate PR, since it doesn't really affect this one.

Reply:
Yes and sure.